edenai_apis/apis/amazon/amazon_image_api.py

```diff
@@ -498,14 +504,56 @@ def image__generation(
         response_body = json.loads(response.get("body").read())
         generated_images = []
         for image in response_body["images"]:
-            base64_bytes = image.encode('ascii')
+            base64_bytes = image.encode("ascii")
             image_bytes = BytesIO(base64.b64decode(base64_bytes))
             resource_url = upload_file_bytes_to_s3(image_bytes, ".png", USER_PROCESS)
             generated_images.append(
-                GeneratedImageDataClass(image = image, image_resource_url=resource_url)
+                GeneratedImageDataClass(image=image, image_resource_url=resource_url)
             )
 
         return ResponseType[GenerationDataClass](
             original_response=response_body,
             standardized_response=GenerationDataClass(items=generated_images),
-        )
\ No newline at end of file
+        )
+
+    def image__embeddings(
+        self,
+        file: str,
+        file_url: str = "",
```

Comment:
I would have set file_url at the end, as it's the parameter the less likely to be set by a user if he uses our package. But it's also a small detail